uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/JobRequestHelper.java

```diff
@@ -115,6 +115,9 @@ public JobResult cloneProject(final String jobId,
         if (cloneProjectRequest.getPassword() != null && !"".equals(cloneProjectRequest.getPassword())) {
             configuration.setPassword(cloneProjectRequest.getPassword());
         }
+
+        configuration.setInit(false);
+        configuration.setMirror(false);
```

Comment:
I copied this setup from the UI repository import. So we already use them in other location.

Reply:
import here = clone